core/src/test/java/org/elasticsearch/index/mapper/DynamicMappingDisabledIT.java

```diff
@@ -0,0 +1,29 @@
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.test.ESIntegTestCase;
+
+import java.io.IOException;
+
+/**
+ * Created by davidgalbraith on 12/17/15.
```

Comment:
We greatly appreciate the contribution, but we do not place attribution in our source code like this. :)